salvador/APIData.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  apiData.swift
+//  salvador
+//
+//  Created by Geury Roustand on 06.02.23.
+//
+
+import Foundation
+
+struct APIData: Codable {
+    
+   struct urlImageResponse:Codable {
```

Comment:
I thought a bit more about model names when I reviewed Leonits code...
Here is my suggestion: `APIData` could be named `ImageDataResponse` because it is the actual response model that contains everything.
`URLImageResponse` could be simply named `ImageURL`.
Does that make sense? WDYT?

Furthermore, there is a small formatting issue with a missing space `urlImageResponse: Codable`.
small tip:  you can auto-format selected code by pressing `control + i`

Reply:
Yes, u are right makes more sense :) I also fixed the formating, thank u for the tip.